jujugui/static/gui/src/app/views/topology/relation.js

```diff
@@ -359,6 +359,26 @@ YUI.add('juju-topology-relation', function(Y) {
             // Style relation lines differently depending on status.
             return 'relation ' + d.aggregatedStatus;
           });
+      enter.append('circle')
```

Comment:
I'm assuming that these are the little knobs on the end of the lines? Can you add a comment to this effect